### ISSUE TITLE
Update contributing guide

### ISSUE THREAD

**Issue by sourabhtk37** [sourabhtk37/CodeShare#31]
Update contributing guide to have more details as to how to report any issues related to the project.

Better add a `.github` file to the project which have `ISSUE_TEMPLATE` and `PULL_REQUEST_TEMPLATE`.

**Comment by zoyron** [comment]
should i add this .github file thing to the contributing.md?

**Comment by nishutosh** [comment]
Yes you may, please read this [.github](https://github.com/rhysd/dot-github) .

**Comment by sourabhtk37** [comment]
You should research a bit about as to this "thing" `.github` folder is and what are it's uses. @zoyron 
Also @nishutosh that's a generator what's the use of that here?

**Comment by sourabhtk37** [comment]
@zoyron Here are some articles:
[Writing your contributing guidelines](https://opensource.guide/starting-a-project/#writing-your-contributing-guidelines)
[Issue and Pull Request templates](https://github.com/blog/2111-issue-and-pull-request-templates)

**Comment by nishutosh** [comment]
Template Format example at [.github](https://github.com/rhysd/dot-github) and this repo has the link you mentioned above. Btw it would be convenient  to use this generator 😃

**Comment by sourabhtk37** [comment]
@zoyron figured it out?
